src/shell.js

```diff
@@ -15,7 +15,7 @@
 // can continue to use Module afterwards as well.
 var Module;
 #if USE_CLOSURE_COMPILER
-if (!Module) Module = eval('(function() { try { return {{{ EXPORT_NAME }}} || {} } catch(e) { return {} } })()');
+if (!Module) Module = eval('(function() { try { return Module || {} } catch(e) { return {} } })()');
```

Comment:
this does look like the right change for the MODULARIZE case. however, EXPORT_NAME might be used without MODULARIZE, in which case, we don't create a function to wrap everything, but we would use EXPORT_NAME instead of Module. In that case, I think it is correct the old way, we want to use the EXPORT_NAME variable (to have Module point to it, so that whatever EXPORT_NAME contained before is still used).

Whereas in MODULARIZE, Module is the correct name because it's the only name in the function that wraps everything.

Does that sound correct?

If so, then we could either do one of two things on this line, or in the code for MODULARIZE, instead of `function (Module)` we could do `function ({{{ EXPORT_NAME }}})`. I tend to the second option, I think - thoughts?